src/components/IssueController.jsx

```diff
@@ -61,7 +61,7 @@ export default class IssueController extends FalcorController {
           //console.log("Category: " + JSON.stringify(category));
           return(
             <div key={category} className="issue__category">
-              <h2 className="section-header">{category}</h2>
+              <h2 className="section-header">{category.replace('-', ' ')}</h2>
```

Comment:
Probably we should have category and category slug as two fields. Then we can pass them both along.